sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/testinterface/ComRunner.scala

```diff
@@ -34,14 +34,18 @@ class ComRunner(bin: File,
       logger.info(s"Starting process '$bin' on port '$port'.")
       Process(bin.toString +: port.toString +: args, None, envVars.toSeq: _*) ! logger
       running = false
+      socket.close()
+      serverSocket.close()
```

Comment:
You must always `close` in finally clause. Otherwise if exception happens, you leak the resource.